docs/src/techpapers/netcdf_io.rst

```diff
@@ -0,0 +1,116 @@
+.. _netcdf_io:
+
+.. testsetup:: chunk_control
+
+    import iris
+    from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+
+    from pathlib import Path
+    import shutil
+    import tempfile
+
+    tmp_dir = Path(tempfile.mkdtemp())
+    tmp_filepath = tmp_dir / "tmp.nc"
+
+    cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+    iris.save(cube, tmp_filepath, chunksizes=(120, 37, 49))
+
+
+.. testcleanup:: chunk_control
+
+    shutil.rmtree(tmp_dir)
+
+
+=============================
+NetCDF I/O Handling in Iris
+=============================
+
+This document provides a basic account of how NetCDF files work within Iris.
+
+This document is still a work in progress, so might include blank or unfinished sections,
+watch this space!
+
+
+Chunk Control
+--------------
+
+Default Chunking
+^^^^^^^^^^^^^^^^
+
+Chunks are, by default, optimised by iris on load. This will automatically
+decide the best chunksize for your data without any user input.
+
+.. doctest:: chunk_control
+
+    >>> # DEFAULT CHUNKING
+    >>> cube = iris.load_cube(tmp_filepath)
+    >>>
+    >>> print(cube.lazy_data().chunksize)
+    (240, 37, 49)
```

Comment:
I'd like to see the `shape` of the data too, to demonstrate that the default `chunksize` isn't just the whole file - there's a lot of intelligence going on there.

Reply:
EDIT: I see that this _is_ the shape of the whole file 😂

Could put something in the setup to either drop Dask's chunk bytes, or could use `merge` on the cube to artificially create a 4th dimension large enough to require chunking.